src/Resource.cs

```diff
@@ -26,6 +23,10 @@ public Resource_info(Vessel v, string res_name)
 							amount += r.amount;
 							capacity += r.maxAmount;
 						}
+#if DEBUG
```

Comment:
Could you give this it's own Directive 😉 something like RES_DEBUG

Reply:
Sorry, but I'm not sure how I do it, I can search to be sure, but if you can do this or send me the instructions, I'll do it.